data/json/items/armor/helmets.json

```diff
@@ -306,6 +306,36 @@
     ],
     "melee_damage": { "bash": 2 }
   },
+  {
+    "id": "helmet_soviet_tanker",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "soviet tanker helmet" },
+    "description": "A simple helmet meant to protect the head from fragmentation and impacts.  Its simple steel construction renders it outdated by modern military standards.",
+    "price": 10000,
+    "price_postapoc": 1200,
+    "weight": "950 g",
+    "volume": "600 ml",
+    "symbol": "[",
+    "looks_like": "helmet_army",
+    "color": "green",
+    "warmth": 30,
+    "flags": [ "VARSIZE", "STURDY", "PADDED" ],
+    "armor": [
+      {
+        "layers": [ "NORMAL" ],
+        "encumbrance": 10,
+        "covers": [ "head" ],
+        "specifically_covers": [ "head_crown", "head_forehead", "head_ear_l", "head_ear_r" ],
+        "coverage": 100,
+        "material": [
+          { "type": "canvas", "thickness": 1 },
```

Comment:
you have two canvas and no steel. Is that intended?

Reply:
Yes, the second canvas is meant to represent the foam "casings", and the absence of steel is also intended, this is a soft helmet and unless I'm mistaken there is no steel plating in it.

Reply:
The TSh-4 is a soft, padded leather/canvas (on later models) helmet. Doesn't contain steel, but were meant to protect from the tank's steel interior. The description is from `helmet_army_outdated` and probably an oversight.

Reply:
Ok so then the description needs to stop saying "simple steel"
```
"A simple helmet meant to protect the head from fragmentation and impacts.  Its simple steel construction renders it outdated by modern military standards.",
```

Reply:
I should have went to sleep **before** posting this looks like. Give me a moment.

Reply:
> Ok so then the description needs to stop saying "simple steel"
> 
> ```
> "A simple helmet meant to protect the head from fragmentation and impacts.  Its simple steel construction renders it outdated by modern military standards.",
> ```

There you go, I updated the description to what it should be. Do tell me if the formulation feels weird.